formula/src/test/java/com/instacart/formula/FormulaRuntimeTest.kt

```diff
@@ -660,6 +660,8 @@ class FormulaRuntimeTest(val runtime: TestableRuntime, val name: String) {
             }
     }
 
+    // TODO: does this
+    @Ignore
```

Comment:
This behavior changes because all events are added to the queue and processed in order. So `increment` will be processed before `toggleCallback`.